External/Plugins/HaXeContext/Completion/HaxeComplete.cs

```diff
@@ -258,7 +258,6 @@ HaxeCompleteStatus ParseLines(string lines)
                         Errors = lines;
                         return HaxeCompleteStatus.ERROR;
                     }
-                    break;
```

Comment:
I know it's not needed, but feels strange to remove it to me. Anyway, not really important I guess.